mlrun/api/api/utils.py

```diff
@@ -679,6 +679,13 @@ def _submit_run(
         # fn.spec.rundb = "http://mlrun-api:8080"
         schedule = data.get("schedule")
         if schedule:
+            if data.get("overwrite_schedule"):
+                get_scheduler().delete_schedule(
+                    db_session,
+                    task["metadata"]["project"],
+                    task["metadata"]["name"],
+                )
+                db_session.commit()
```

Comment:
inside the `delete_schedule` method there is also a `commit()` happening, so we need to think is it needed to commit here again.

Reply:
It didn't work before I added this `commit()`, @Tankilevitch your opinion?

Reply:
Are you sure? After delete/create/update we are committing and it supposed to clean the "cached" entries.

Reply:
triple-checking it

Reply:
deleted.